docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -33,9 +33,9 @@ Test API provides a set of **actions** you can use to interact with the page.
 * [Resize Window](#resize-window)
 * [Wait](#wait)
 
-They are implemented as methods in the [test controller](../../reference/test-api/testcontroller/README.md) object. You can call them in a chained fashion.
+[Test controller](../../reference/test-api/testcontroller/README.md) object includes them as its methods. You can call them in a chained fashion.
```

Comment:
`[Test controller](../../reference/test-api/testcontroller/README.md) object includes them as its methods.`
Specify that "them" refers to "these objects"

`You can call them in a chained fashion.`
What do you mean by "chained fashion"?

Reply:
1. Is that better:
> [Test controller](../../reference/test-api/testcontroller/README.md) object includes actions as its methods

2. When we call actions in code, we chain them one onto the other with a dot:
Item.**hover**.**click**

Reply:
Thank you.

1) Test controller object includes actions as its methods
Please add an article to "test controller object". (e.g., "The test controller object")

2) Can we say:
`You can chain these actions when you call them in code.`
OR
`You can chain these actions in code.`

Reply:
1. Done

2. Works like a charm, thanks